.github/workflows/unit_tests.sh

```diff
@@ -76,7 +76,7 @@ for phase in "${PHASES[@]}"; do
             find . -type f -name meson.build -exec sed -i '/install_tag/d' '{}' '+'
             MESON_ARGS+=(--fatal-meson-warnings)
             run_meson -Dnobody-group=nogroup --werror -Dtests=unsafe -Dslow-tests=true -Dfuzz-tests=true "${MESON_ARGS[@]}" build
-            ninja -C build -v
+            meson compile -C build -v
```

Comment:
I don't think we should do this. This is very sad that meson has dug itself into this mess, but things are as they are. So until this is fixed in meson, let's keep using the command that works.

```console
$ meson compile -C build -v
ERROR: Coredata file '/home/zbyszek/src/systemd-work/build/meson-private/coredata.dat' references functions or classes that don't exist. This probably means that it was generated with an old version of meson. Try running from the source directory meson setup /home/zbyszek/src/systemd-work/build --wipe
$ ninja -C build -v
WARNING: Regenerating configuration from scratch.
...
(works)
```

Reply:
Unless you want to support build backends other than `ninja` I don't think using `meson compile` has any advantage. In the best case, it makes compilation slower because the meson startup time is significant compared to the ninja startup time.

Reply:
What exactly is the mess here anyway? `ninja -C build/ -v` will only regenerate the configuration from scratch when explicitly told to do so because a ninja dependency on e.g. `meson.build` forced a refresh. `meson compile` doesn't try to do any such thing.

None of this should be relevant unless you've updated your meson version and it's no longer the same version as the one you configured the build with.